src/pages/LoginAndRegister/RegisterSocial.vue

```diff
@@ -10,8 +10,8 @@
 			<form
 				id="registerSocialTermsForm"
 				class="promptForm tw-text-left"
-				action="."
-				@submit.prevent.stop="postRegisterSocialForm"
+				:action="`https://${$appConfig.auth0.domain}/continue?state=${$route.query.state}`"
```

Comment:
Does the query on this url still need the `newsConsent` query param used elsewhere or is that just a fallback now?